packages/mds-jurisdiction-service/migrations/1582294819607-CreateJurisdictionsTable.ts

```diff
@@ -21,13 +21,19 @@ export class CreateJurisdictionsTable1582294819607 implements MigrationInterface
 
   public async up(queryRunner: QueryRunner): Promise<void> {
     await queryRunner.query(
-      `CREATE TABLE "jurisdictions" ("id" bigint GENERATED ALWAYS AS IDENTITY, "recorded" bigint NOT NULL, "jurisdiction_id" uuid NOT NULL, "agency_key" character varying(63) NOT NULL, "versions" json NOT NULL, CONSTRAINT "jurisdictions_pkey" PRIMARY KEY ("jurisdiction_id"))`,
+      `CREATE TABLE IF NOT EXISTS "jurisdictions" ("id" bigint GENERATED ALWAYS AS IDENTITY, "recorded" bigint NOT NULL, "jurisdiction_id" uuid NOT NULL, "agency_key" character varying(63) NOT NULL, "versions" json NOT NULL, CONSTRAINT "jurisdictions_pkey" PRIMARY KEY ("jurisdiction_id"))`,
```

Comment:
It looks like TypeORM has more programmatic ways of doing this (i.e. calling `createTable` and passing in the desired columns). Is that incompatible with using a separate migration table per service?

Reply:
It does, but I prefer using the CLI to generate migrations based on comparing the database schema to the model.

Reply:
It does, but I prefer using the CLI to generate migrations based on comparing the database schema to the model.

Reply:
They also have a programmatic interface (which the CLI uses) to generate migrations against an existing DB too. I agree we should avoid doing things like `createTable` — that feels very similar to what we’re doing currently with mds-db.